src/rand.rs

```diff
@@ -385,17 +385,16 @@ mod darwin {
     #[repr(C)]
     struct SecRandomRef([u8; 0]);
 
+    // https://developer.apple.com/documentation/security/ksecrandomdefault
+    // says "This constant is a synonym for NULL."
+    const kSecRandomDefault: *const SecRandomRef = core::ptr::null();
```

Comment:
I think the UB is fixed but the types here are misleading. SecRandomRef refers to a different thing in Security Framework than here.

```suggestion
    #[repr(C)]
    struct __SecRandom([u8; 0]);
    type SecRandomRef = *const __SecRandom;

    // https://developer.apple.com/documentation/security/ksecrandomdefault
    // says "This constant is a synonym for NULL."
    const kSecRandomDefault: SecRandomRef = core::ptr::null();
```

Reply:
Thanks! I made those changes plus also changed the declaration of of `SecRandomCopyBytes` accordingly.